compiler/src/Language/Bond/Codegen/Java/Util.hs

```diff
@@ -9,6 +9,7 @@ module Language.Bond.Codegen.Java.Util
     , modifierConstantName
     , isPrimitiveNonEnumBondType
     , isPrimitiveBondType
+    , isJavaPrimitive
```

Comment:
I think that's redundant - it's defined on a `Type`, and it's used in places where it's clearly acting on a `Type`. I'd like to keep it like this.